src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-events.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: 'OpenTelemetry in the UI: Events'
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+metaDescription: You can query experimental OpenTelemetry events in New Relic.
+---
+
+import otelEventsQueryScreenshot from 'images/otel-events.png'
+
+
+<Callout title="Limited preview">
+  If you sign up for the limited preview of the Events API, you can query OpenTelemetry events in New Relic. Let us know in our [CNCF Slack channel, #otel-newrelic](https://cloud-native.slack.com/archives/C024DRQ63UP) if you'd like to participate.
```

Comment:
Is this an OpenTelemetry feature preview and not our preview? In that case I'd make that more clear either way. if it's our preview, we should just say 'preview' and not 'limited preview'. If this is first place we introduce this, we should probably explain 'This feature is in preview.' or whatever.